grids.js

```diff
@@ -52,11 +52,14 @@
             cols = 0;
         this.each(function(i, elem) {
```

Comment:
If we used $tiles.each, would that mean we wouldn't have to check if the item is visible inside the loop?